.lintrunner.toml

```
@@ -940,6 +940,25 @@ command = [
     '@{{PATHSFILE}}'
 ]
 
+[[linter]]
+code = 'CONTEXT_DECORATOR'
+include_patterns = [
+    'torch/**',
+]
+command = [
+    'python3',
+    'tools/linter/adapters/grep_linter.py',
+    '--pattern=@.+(dynamo_timed)',
```

```suggestion
    '--pattern=@.*(dynamo_timed)',
```

Or you specifically don't want to capture plain `@dynamo_timed` ?

---

I want to catch @torch._dynamo.utils.dynamo_timed() too

---

Right, so you want the `*` version, not `+` ?

---

oh whoops you right